pkg/consolegraphql/agent/agent.go

```diff
@@ -134,6 +134,7 @@ func (aad *amqpAgentDelegate) doCollect() error {
 		amqp.ConnSASLXOAUTH2("unused", aad.bearerToken, amqpOverrideSaslFrameSize),
 		amqp.ConnServerHostname(aad.host),
 		amqp.ConnProperty("product", "console-server"),
+		amqp.ConnConnectTimeout(time.Second * 10),
```

Comment:
Should this be tunable as well? I remember we had to do that for the standard-controller <-> router connection .